.github/workflows/ci.yml

```diff
@@ -1,6 +1,18 @@
 name: CI
 
 on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version of Kotlin binary to publish to TBD Artifactory. For example "1.0.0-SNAPSHOT". If not supplied, will default to "commit-$shortSHA-SNAPSHOT" where $shortSHA is the shortform commit SHA. Must end in "-SNAPSHOT".'
+        required: false
+        default: "0.0.0-SNAPSHOT"
+  workflow_call:
+    inputs:
+      version:
+        description: 'Version of Kotlin binary to publish to TBD Artifactory. For example "1.0.0-SNAPSHOT". If not supplied, will default to "commit-$shortSHA-SNAPSHOT" where $shortSHA is the shortform commit SHA. Must end in "-SNAPSHOT".'
```

Comment:
wrt `Must end in "-SNAPSHOT".` -- I don't see any check to ensure it ends with `-SNAPSHOT` in the `resolve_version` step, but I'm assuming this is fine since `tbdex-rs` is also like that.

Reply:
Good catch. Fixed in latest commit!

Reply:
Also opened this https://github.com/TBD54566975/tbdex-rs/issues/112